game/guitargraph.hh

```diff
@@ -121,14 +123,14 @@ class GuitarGraph: public InstrumentGraph {
 	void drawNeckStuff(double time);  ///< Anything in neck coordinates
 	void drawNotes(double time);  ///< Frets etc.
 	void drawBar(double time, float h);
-	void drawNote(unsigned fret, Color, float tBeg, float tEnd, float whammy = 0, bool tappable = false, bool hit = false, double hitAnim = 0.0, double releaseTime = 0.0);
-	void drawDrumfill(float tBeg, float tEnd);
+	void drawNote(unsigned fret, Color, double tBeg, double tEnd, float whammy = 0.0f, bool tappable = false, bool hit = false, double hitAnim = 0.0, double releaseTime = 0.0);
+	void drawDrumfill(double tBeg, double tEnd);
```

Comment:
Again, don't understand why this became double

Reply:
It's times. Float is not good enough for time.